internal/config/mount_config.go

```diff
@@ -24,9 +24,29 @@ type LogConfig struct {
 	FilePath string      `yaml:"file-path"`
 }
 
+type FileCacheConfig struct {
+	Path                    string `yaml:"path"`
+	Size                    int64  `yaml:"size-mb"`
+	TTL                     int64  `yaml:"ttl-sec"`
+	CacheFileForRandomReads bool   `yaml:"cache-file-for-random-reads"`
+}
+
+type MetadataCacheConfig struct {
+	Capacity int64 `yaml:"capacity"`
+	TTL      int64 `yaml:"ttl-sec"`
+}
+
+type TypeCacheConfig struct {
+	TTL              int64 `yaml:"ttl-sec"`
+	CacheNonexistent bool  `yaml:"cache-nonexistent"`
```

Comment:
We want to avoid changes around flags as much as possible. Until we decide on a plan for type-cache dont move around anything